# 状态标志位
bool is_connected   # 连接状态
bool is_locked      # 锁定状态
bool is_syncronised     # 同步状态
bool sync_completed #完成同步状态
bool is_recording   # 录制状态